### PR TITLE
Feature/jsx css rework

### PR DESCRIPTION
**Pull request by Creepypoke**

### CODE REVIEW

File: src/components/base/JRaisedButton/JRaisedButton.js [hunk 1]
```diff
@@ -1,119 +1,75 @@
...
-      <div
-        onMouseEnter={handle(this.onHover)(true)}
-        onMouseLeave={handle(this.onHover)(false)}
-        onClick={isDisabled ? undefined : onClick}
+      <button
+        type='button'
+        {...omittedProps}
         className={buttonClassName}
+        onClick={this.props.onClick}
+        disabled={isDisabled}
```
Comment (IViol) [inline src/components/base/JRaisedButton/JRaisedButton.js]: need to add `disabled` to `omittedProps`

File: src/components/base/JRaisedButton/JRaisedButton.js [hunk 2]
```diff
@@ -1,119 +1,75 @@
...
-  +isLoading: boolean,
-  +isDisabled: boolean,
-|}
+type Props = {
+  children: React$Node,
+  theme: Themes,
+  onClick?: Function,
+  isLoading?: boolean,
+  disabled?: boolean,
+  className?: ?string,
```
Comment (flexxnn) [inline src/components/base/JRaisedButton/JRaisedButton.js]: Should we use `className: string | Object` to pass some Object values to cx too?

File: src/routes/Upgrade/UpgradeView.js [hunk 3]
```diff
@@ -53,6 +51,9 @@ const renderPasswordField = (isInvalidPassword: boolean) => (fieldProps) => {
   return <JInputField {...fieldProps} />
 }
 
+/**
+ * @return {null}
+ */
```
Comment (IViol) [inline src/routes/Upgrade/UpgradeView.js]: ?

File: src/components/base/JRaisedButton/JRaisedButton.js (hunk 2 above)
Comment (Creepypoke) [reply src/components/base/JRaisedButton/JRaisedButton.js]: I guess, we should using cx explicitly in context.

**Pull request merged**